octoprint_dashboard/static/js/dashboard.js

```diff
@@ -969,6 +976,15 @@ $(function() {
             self.layerGraph = new Chartist.Line('.ct-chart');
 
             self.doTempGaugeTicks();
+
+            document.addEventListener("visibilitychange", () => {
+                console.log(document.visibilityState);
+                if (document.visibilityState == 'visible') {
+                    self.isTabVisible(true);
```

Comment:
This should check if the current tab is #tab_plugin_dashboard before enabling the stream again

Reply:
done